index.html

```diff
@@ -5,74 +5,57 @@
-        width: 400px;
-    }
-</style>
-
-<body>
-    <center>
-
-        <h1>Welcome to the 2018 WSB Season!!</h1>
-        <br />
-        <img src="https://bsbproduction.s3.amazonaws.com/portals/10758/logo636199038037105207.png">
-        <img src="images/IMG_2922.gif">
-        <br />
-        <br />
-
-
-        <div class="about">
-            <p>This is where the schedule for the year will be. It is your job to check the site on a weekly basis. You will
-                also be able to send your availabilty here, as well as ask to drop/pick up a shift.
-                <br />
-                <br /> The new schedule will be posted every saturday night.</p>
-        </div>
-
-        <ul>
-            <li>
-                <a href="index.html" id="titles" class="active">
-                    <h3>Home</h3>
-                </a>
-            </li>
-            <li>
-                <a href="WSB_Schedule.html" id="titles">
-                    <h3>Schedule</h3>
-                </a>
-            </li>
-            <li>
-                <a href="WSB_Contacts.html" id="titles">
-                    <h3>Contacts</h3>
-                </a>
-            </li>
-            <li>
-                <a href="WSB_Help.html" id="titles">
-                    <h3>Help</h3>
-                </a>
-            </li>
-            <li>
-                <a href="WSB_Availabilty.html" id="titles">
-                    <h3>Availabilty</h3>
-                </a>
-            </li>
-        </ul>
-
-        <footer>
-            <br /> &copy;GreenMountainWebs 2018
-        </footer>
-
+    <style>
+        body{
+            color: azure;
+        }
+        div.about{
+            background-color: rgba(0,0,0,.3);
+        }
+        
+        h1{
+            font-size: 40px;
+        }
+        p{
+            font-size: 20px;
+            width: 400px;
+        }
+    </style>
+    
+<body><center>
+    
+    <h1>Welcome to the 2018 WSB Season!!</h1>
+    <br />
+      <img src="https://bsbproduction.s3.amazonaws.com/portals/10758/logo636199038037105207.png">
+    <img src="images/IMG_2922.gif">
+    <br />
+    <br />
+    
+  
+    <div class="about">
+    
+    <p>This is where the schedule for the year will be. It is your job to check the site on a weekly basis. You will also be able to send your availabilty here, as well as ask to drop/pick up a shift.
+        <br /><br />
+        The new schedule will be posted every saturday night.</p>
+    </div>
+    
+    <ul>
+            <li><a href="index.html" id="titles" class="active"><h3>Home</h3></a></li>
+        
+           <li><a href="WSB_Schedule.html" id="titles"><h3>Schedule</h3></a></li>
+           
+           <li><a href="WSB_Contacts.html" id="titles"><h3>Contacts</h3></a></li>
+            
+          <li><a href="WSB_Help.html" id="titles"><h3>Help</h3></a></li> 
+            
+            <li><a href="passwordtest.html" id="titles"><h3>Availabilty</h3></a></li>
+            
+            </ul>
```

Comment:
@geodrummer what happened to my formatting? What did you branch off of to create this new branch? Did you fetch and pull before branching?

Reply:
You did not fetch/pull your password branch before branching. I even mentioned it to you bro on #12 
> I also merged js to your pw-protect branch so all my formatting changes should be in that branch. Make sure you pull before you work on it.

Not to be mean but I would erase this branch and start again. Your changes don't seem to be that major. You need to get in the habit of fetching and pulling. I know you did not because you are missing my 1b2cf935747153e2921a1f9c84fd4db2cd4d986f commit. 

Or better yet, before you commit/branch/or do anything in git kraken, make sure your local branch has the latest commit in the branch. You can view all branches here, https://github.com/geodrummer/WSB-Site/branches, and if you click one, you can see all the commits.

Reply:
It is hard to explain with words so I thought I would also provide you with a visual. In my screen shot I have hid the "feature/schedule-backend" branch just so it is out of the way. You can do that by clicking the picture of the eye next to the branch in git kraken. When looking at the graph in git kraken, you need to pay attention to the circles and the lines. The circle is a commit and the line tells you a lot about merges. I have also linked all the commits so you can see what they look like in github. 

 1. This is the commit you started at when working on the #14 and #15 changes. You branched off a commit that wasn't the head of any branch. In your defense, you probably thought that was the head of the js branch. It would have looked like the head if you did not fetch and pull the latest js branches I made (which is the next commit, point 2). So 
essentially by branching off of here, instead of point 2 below, you were saying, "I want all the changes in the js branch except the latest one". 4e64a2f71b126cec4a8069c1d0a12ca8f42c7a70

2. This is the head of the js branch. Branching off of here is what I would have done. Branching off of here is saying "I want all the changes in the js branch, yes all the changes including the latest one.." b112ef31b92948fe9a67d736f71387601d1ee221

3. This is your commit. 149275bf0b822ebe86651ea78361804757cf67b9

4. And then you probably were like oh shit... i forgot to fetch and pull before working.  So you decided to merge js into your issue branch. Again, if you would have just branched off of point 2 above, you would have already had these changes and you wouldn't have had to merge. 6c45643c4ffb1f15ed453b6ced24dfd0c1d08313

Is any of this making sense to you? 

![image](https://user-images.githubusercontent.com/17501276/37232185-8aa7a350-23b3-11e8-9f9d-7242b1a7183b.png)

Reply:
what do you mean about your formatting? i did fetch before i commit. there were conflicts with some of the code from the pervious commit...so maybe i did something i wasnt suppose to do when i was fixing that

Reply:
Yes, that is the problem. You did not fetch/pull before creating your branch. You branched off a commit that wasn't the head of another branch.

Reply:
so how do i fix it? i delete the branch and do it again?

Reply:
Yup, that is my suggestion.  Notice the lines in my screen shot.  You went from 1 to 3.  You should be going from 2 to 3.

Reply:
I can the branch if you are having issues deleting it in git kraken.  I was having issues the other day deleting stuff.

Reply:
yes delete it